src/Component/Symbolizer/LineEditor/LineEditor.tsx

```diff
@@ -82,12 +81,17 @@ export interface LineEditorProps extends Partial<LineEditorDefaultProps> {
 }
 const COMPONENTNAME = 'LineEditor';
 
-export const LineEditor: React.FC<LineEditorProps> = ({
-  locale = en_US.LineEditor,
-  symbolizer,
-  onSymbolizerChange,
-  defaultValues
-}) => {
+export const LineEditor: React.FC<LineEditorProps> = (props) => {
+
+  const composition = useGeoStylerComposition('LineEditor', {});
```

Comment:
You could use `useGeostylerContext` here to also use the locale directly.

Reply:
Sure, I just wanted to keep things separate at first, for better reviews. I will update this in subsequent PRs.